examples/window-app/silabs/src/WindowManager.cpp

```diff
@@ -742,6 +742,7 @@ void WindowManager::ButtonEventHandler(uint8_t button, uint8_t btnAction)
     }
 
     AppTask::GetAppTask().PostEvent(&event);
+
```

Comment:
This file should be removed from the commit

Reply:
done